### PR TITLE
Edit issues templates

### PR DESCRIPTION
**Pull request by marionf**
| Questions         | Answers
| ----------------- | -------------------------------------------------------
| Branch?           | develop
| Description?      | Edit issues templates
| Type?             | Improvement
| Category?         | PM
| BC breaks?        | no
| Deprecations?     | no
| Fixed ticket?     | Fixes #{issue number here}.
| How to test?      | Try to create a bug report and a feature request
| Possible impacts? | Please indicate what parts of the software we need to check to make sure everything is alright.

---
This change is [Reviewable](https://reviewable.io/reviews/prestashop/prestashop/26287)

### CODE REVIEW

**Review by Progi1984 — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/1_bug_report.yml [hunk 1]
```diff
@@ -0,0 +1,65 @@
...
+      description: What went wrong? If applicable, add screenshots or screen recordings to help explain your problem.
+    validations:
+      required: true
+  - type: textarea
+    id: expected-behavior
+    attributes:
+      label: Expected behavior
+      description: Explain what you expected to happen instead.
+    validations:
+      required: false
```
Comment (atomiix) [inline .github/ISSUE_TEMPLATE/1_bug_report.yml]:
```suggestion
      required: false
```

File: .github/ISSUE_TEMPLATE/1_bug_report.yml [hunk 2]
```diff
@@ -0,0 +1,65 @@
...
+      label: Steps to reproduce
+      description: List the exact steps to reproduce the issue
+      placeholder: | 
+        1. Go to '...' 
+        
+        2. Click on '....' 
+        
+        3. Scroll down to '....' 
+        
+        4. See error
```
Comment (atomiix) [inline .github/ISSUE_TEMPLATE/1_bug_report.yml]:
```suggestion
      placeholder: |
        1. Go to '...'

        2. Click on '....'

        3. Scroll down to '....'

        4. See error
```

File: .github/ISSUE_TEMPLATE/2_feature_request.yml [hunk 3]
```diff
@@ -0,0 +1,44 @@
...
+      options:
+        - label: I understand and accept the project's [code of conduct](https://github.com/PrestaShop/PrestaShop/blob/develop/CODE_OF_CONDUCT.md)
+          required: true
+        - label: I have already [searched in existing features request](https://github.com/PrestaShop/PrestaShop/issues?q=is%3Aissue+label%3AImprovement%2CFeature%2CDeveloper+Feature+) and found no previous suggestion of this feature.
+          required: true
+  - type: textarea
+    id: problem
+    attributes:
+      label: Is your feature request related to a problem?
+      description: A clear and concise description of what the problem is.
```
Comment (atomiix) [inline .github/ISSUE_TEMPLATE/2_feature_request.yml]:
```suggestion
      description: A clear and concise description of what the problem is.
```

File: .github/ISSUE_TEMPLATE/2_feature_request.yml [hunk 4]
```diff
@@ -0,0 +1,44 @@
...
+      placeholder: I'm always frustrated when...
+    validations:
+      required: false
+  - type: textarea
+    id: solution
+    attributes:
+      label: Describe the solution you'd like
+      description: A clear and concise description of what you'd want to happen.
+    validations:
+      required: true
```
Comment (atomiix) [inline .github/ISSUE_TEMPLATE/2_feature_request.yml]:
```suggestion
      required: true
```

File: .github/ISSUE_TEMPLATE/2_feature_request.yml [hunk 5]
```diff
@@ -0,0 +1,44 @@
...
+      description: A clear and concise description of what you'd want to happen.
+    validations:
+      required: true      
+  - type: textarea
+    id: alternative
+    attributes:
+      label: Alternatives you've considered
+      description: A clear and concise description of any alternative solutions or features you've considered.
+    validations:
+      required: false
```
Comment (atomiix) [inline .github/ISSUE_TEMPLATE/2_feature_request.yml]:
```suggestion
      required: false
```

**Review by atomiix — DISMISSED** [review]

File: .github/ISSUE_TEMPLATE/1_bug_report.yml [hunk 6]
```diff
@@ -0,0 +1,65 @@
+name: 🐛 Bug Report
+description: Report an issue in PrestaShop or one of the native modules. DO NOT disclose security issues here, contact security@prestashop.com instead!
```
Comment (boubkerbribri) [inline .github/ISSUE_TEMPLATE/1_bug_report.yml:2]:
```suggestion
description: Report an issue in PrestaShop or one of the native modules. DO NOT disclose security issues here, contact security@prestashop.com instead!
labels: ["Bug"]
```

File: .github/ISSUE_TEMPLATE/2_feature_request.yml [hunk 7]
```diff
@@ -0,0 +1,44 @@
+name: 🚀 Feature Request
+description: Suggest an idea for this project
```
Comment (boubkerbribri) [inline .github/ISSUE_TEMPLATE/2_feature_request.yml:2]:
```suggestion
description: Suggest an idea for this project
labels: ["Feature"]
```

**Review by boubkerbribri — COMMENTED** [review]
WDYT of adding label `Bug` or `Feature` automatically depending on the type of the issue?

**Comment by marionf** [comment]
Good idea @boubkerbribri 👍

File: .github/ISSUE_TEMPLATE/1_bug_report.yml [hunk 8]
```diff
@@ -0,0 +1,66 @@
...
+labels: ["Bug"]
```
Comment (Progi1984) [inline .github/ISSUE_TEMPLATE/1_bug_report.yml]:
```suggestion
labels: [ "Bug" ]
```

File: .github/ISSUE_TEMPLATE/2_feature_request.yml [hunk 9]
```diff
@@ -0,0 +1,45 @@
...
+labels: ["Feature"]
```
Comment (Progi1984) [inline .github/ISSUE_TEMPLATE/2_feature_request.yml]:
```suggestion
labels: [ "Feature" ]
```

**Review by Progi1984 — COMMENTED** [review]
@marionf I have brought you two contributions. That fixes YAMLLint errors

**Review by Progi1984 — APPROVED** [review]

**Comment by atomiix** [comment]
I think we already have `Bug fix` and `New Feature` added automatically thanks to prestonBot according to what has been filled in the PR description table?

**Comment by marionf** [comment]
For PRs yes but for the issues, it's actually done manually while reproducing the issue

**Comment by atomiix** [comment]
You're right ! 🤦‍♂️

**Review by atomiix — APPROVED** [review]

**Review by matthieu-rolland — APPROVED** [review]
tested and approved 👍

**Comment by matthieu-rolland** [comment]
thank you @marionf !

**Pull request merged**